app/controllers/api/series_controller.rb

```diff
@@ -3,6 +3,8 @@
 class Api::SeriesController < Api::BaseController
   include ApiSearchable
 
+  before_filter :authenticate_user!, only: :create
```

Comment:
Update/delete don't have the same problems?

Also wondering about the other public controllers (stories, etc).  What would happen if this before filter just moved to the base controller itself?  (Since we theoretically don't allow any non-auth'd POST/PUT/DELETE).